services/movie.py

```diff
@@ -0,0 +1,48 @@
+from django.db.models import QuerySet
+
+from db.models import Movie
+
+
+def get_movies(
+        genres_ids: list[int] = None,
+        actors_ids: list[int] = None
+) -> QuerySet[Movie] | Movie:
```

Comment:
Change your type annotation, there is no case when a single `Movie` will be returned, it will be always a `QuerySet` in this function.